apps/microtvm/zephyr/template_project/boards.json

```diff
@@ -23,6 +23,14 @@
         "vid_hex": "",
         "pid_hex": ""
     },
+    "mps3_an547": {
+        "board": "mps3_an547",
+        "model": "mps3_an547",
+        "is_qemu": true,
+        "fpu": false,
+        "vid_hex": "",
+        "pid_hex": ""
```

Comment:
Can you add "note" here and mention why `fpu` is set to false?
something like this: https://github.com/apache/tvm/blob/main/apps/microtvm/arduino/template_project/boards.json#L15